examples/chat/tests/chat.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { test } from "@playwright/test";
+import { LoginPage } from "./pages/LoginPage";
+import { ChatPage } from "./pages/ChatPage";
+
+test("chat between two users", async ({ page }) => {
+    const loginPage = new LoginPage(page);
+
+    const mario = "S. Mario";
+    const luigi = "Luigi";
+
+    await loginPage.goto();
+    await loginPage.fillUsername(mario);
+    await loginPage.signup();
+
+    const chatPage = new ChatPage(page);
+
+    const message1ByMario = "Hello Luigi, are you ready to save the princess?"
+
+    await chatPage.sendMessage(message1ByMario);
+    await chatPage.expectMessageRow(message1ByMario);
+
+    const roomURL = page.url();
+
+    await chatPage.logout();
+
+    await loginPage.expectLoaded();
+
+    await loginPage.fillUsername(luigi);
+    await loginPage.signup();
+
+    await page.goto(roomURL);
+
+    await chatPage.expectMessageRow(message1ByMario);
+
+    const message2ByLuigi = "No, I'm not ready yet. I'm still trying to find the key to the castle."
+
+    await chatPage.sendMessage(message2ByLuigi);
+    await chatPage.expectMessageRow(message2ByLuigi);
+
+    // This breaks with (Error: Error reading from peer storage, handling msg)
+    // await chatPage.logout();
+    // await loginPage.loginAs(mario);
+
+    // await chatPage.expectLoaded();
+
+    // await page.goto(roomURL);
+
+    // await chatPage.expectMessageRow(message1ByMario);
+    // await chatPage.expectMessageRow(message2ByLuigi);
```

Comment:
I've not been able to pinpoint the reason of the crash, but this looks a bad one.

In pratice the message sent by "Luigi" can't be ready by anyone else 🤔

Reply:
Loading another chat room also works.

So something within the shared chat room gets corrupted and isn't stored correctly on IndexedDB